draft/spec/index.html

```diff
@@ -180,24 +180,30 @@ <h2>Introduction</h2>
     <section id="need">
         <h2>Need</h2>
         <p>
-           The OCFL initiative arose from a need to have well-defined application-independent file management within
-           digital repositories.
+          The OCFL initiative arose from a need to have well-defined application-independent file management within
+          digital repositories.
         </p>
         <p>
-           A general observation is that the contents of a digital repository &#8212; that is, the digital files and
-           metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
-           accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
-           applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
-           Thus, transitions between application-specific methods of file management to support software upgrades and
-           replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
-           in support of the short-term, ephemeral software.
+          A general observation is that the contents of a digital repository &#8212; that is, the digital files and
+          metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
+          accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
+          applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
+          Thus, transitions between application-specific methods of file management to support software upgrades and
+          replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
+          in support of the short-term, ephemeral software.
         </p>
         <p>
-           By providing a specification for the file and directory layout on disk or in an object store, OCFL is an
-           attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
-           specification, conforming applications will natively "understand" the underlying file structure without
-           needing to first transition these contents to their own format.
+          By providing a specification for the file and directory layout on disk or in an object store, the OCFL is an
+          attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
+          specification, conforming applications will natively "understand" the underlying file structure without
+          needing to first transition these contents to their own format.
          </p>
+        <p>
+          While digital objects are largely stable, there is also a need to track versions of these objects. These
```

Comment:
"digital objects are largely stable" does not seem to be a true statement to me in the general case. It seems that it would be true to for the archival use case so we could say:

> While archival digital object are expected to be largely stable, ...

but we haven't restricted OCFL to this use case elsewhere. In fact, later we say:

> A central feature of the OCFL specification is support for versioning. This recognizes that digital objects will change over time, through new requirements, fixes, updates, or format shifts. The specification takes no position on what constitutes a version or a versionable action, but it is recommended that implementers have a clear position on this within their local storage policies.

So perhaps we should just say:

> While in many applications digital objects are largely stable, there is also a need to track versions of these objects. 

and then remove the following sentence which is too limited to the archival case and not aligned with the later statement about possible reasons for versions.

Reply:
We do start the "Need" section with:

> A general observation is that the contents of a digital repository -- that is, the digital files and metadata that an institution might wish to manage -- are largely stable.

Reply:
So I was trying to work within that context for the digital object, since it follows that if a repository is largely stable, the objects in that repository must also be largely stable.

Reply:
I still feel that the earlier comment is a bit of an over statement, but it is a nice motivation for file and directory layout. Perhaps the key distinction here is that the **files** are largely stable (hence it is sensible to do deltas by file) but **objects** are more frequently added and added to, although we also support arbitrary updates. I'll make a new suggestion in the main thread as it touches on all three of the bits I commented on...